config-connector/solutions/iam/helm/pubsub-topic/README.md

```diff
@@ -0,0 +1,95 @@
+# Pub/Sub Topic
+
+==================================================
+
+## NAME
+
+  pubsub-topic
+
+## SYNOPSIS
+
+  
+  Config Connector compatible YAML files to create a Pub/Sub topic in your desired project, and grant a specific member a role (default to roles/pubsub.editor) for accessing the Pub/Sub topic that just created
+
+## CONSUMPTION
+
+  1. Clone GoogleCloudPlatform/cloud-foundation-toolkit repository:
+
+      ```bash
+      git clone https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git
+      ```
+
+  1. Go to the service account folder:
+
+      ```bash
+      cd cloud-foundation-toolkit/config-connector/solutions/iam/helm/pubsub-topic
+      ```
+
+## REQUIREMENTS
+
+1. GKE Cluster with Config Connector and [Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#enable_workload_identity_on_a_new_cluster).
+1. [Helm](../../../README.md#helm)
```

Comment:
Could you also add the requirements about IAM permissions and API enablement for this solution? Details can be found [here](https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/tree/master/config-connector/solutions/iam/kpt/pubsub-topic#requirements).

Reply:
Thanks for the update! Just one more requirement you also want to add:
````
  1. The "cnrm-system" service account assigned with either `roles/pubsub.admin` or `roles/owner` in the
  project managed by Config Connector
````